### ISSUE TITLE
Cryptococcus NTR: resistance to starvation phenotype

### ISSUE THREAD

**Issue by Achchuthan** [microbialphenotypes/OMP-ontology#136]
NTR

1)	resistance to starvation phenotype
APO cross-refs:
id: APO:0000091
name: starvation resistance
namespace: observable
alt_id: YPO:0000091
def: "The level of resistance to starvation conditions." [SGD:mcc]
subset: AspGD
subset: CGD
subset: CryptoGD
subset: SGD
synonym: "resistance to starvation" EXACT []
is_a: APO:0000080 ! stress resistance
2)	altered resistance to starvation (is_a resistance to starvation phenotype)
3)	increased resistance to starvation (is_a altered resistance to starvation)
4)	decreased resistance to starvation (is_a altered resistance to starvation)

**Comment by dianeoinglis** [comment]
Starvation can be used for signalling a change in growth conditions. I would not call this a "resistance" phenotype. The APO terms are pretty much entirely based on S. cerevisiae that responds to almost anything as a "stress."

Diane

**Comment by Achchuthan** [comment]
I agree we should change it from "resistance" to "growth under starvation".

**Comment by dianeoinglis** [comment]
Great. Sounds good.

Diane
